### PR TITLE
Cleanup AppVeyor Artifact code

### PR DESCRIPTION
**Pull request by EwoutH**
No change in functionality, just a little better readable.

### CODE REVIEW

**Comment by LordNyriox** [comment]
@EwoutH:  Unfortunately, you changed the way the AppVeyor artifacts are stored. 

x64 Build from your PR:  
[<https://ci.appveyor.com/project/donho/notepad-plus-plus/builds/22004362/job/xxoy1oj90h2dqxl6/artifacts>] 

x64 Build from master:  
[<https://ci.appveyor.com/project/donho/notepad-plus-plus/builds/22041162/job/3dw7ec02gvk2hp63/artifacts>] 

Note the difference in filenames.

File: appveyor.yml [hunk 1]
```diff
@@ -28,25 +28,8 @@ build_script:
...
-
-        if ($env:PLATFORM -eq "Win32" -and $env:CONFIGURATION -eq "Unicode Debug") {
-            Push-AppveyorArtifact "PowerEditor\visual.net\Unicode Debug\Notepad++.exe" -FileName "$nppFileName"
-        }
-
-        Push-AppveyorArtifact "scintilla\bin\SciLexer.dll" -FileName "SciLexer.$env:PLATFORM.$env:CONFIGURATION.dll"
+artifacts:
+   - name: "Notepad++.$(platform).$(configuration).exe"
+     path: \**\Notepad++.exe
+   - name: "SciLexer.$env:PLATFORM.$env:CONFIGURATION.dll"
```
Comment (LordNyriox) [inline appveyor.yml]: @EwoutH:  Also, you need to replace `env:PLATFORM` with `(platform)`, et cetera, in SciLexer.dll as well.

**Comment by EwoutH** [comment]
Well spotted, will fix in a few hours

**Comment by LordNyriox** [comment]
@EwoutH:  FYI, I just glanced through the `appveyor.yml` documentation. 

It does not look like the `artifacts:` group can control actual filenames.

**Comment by donho** [comment]
Let's keep the current working codes.

**Pull request closed without merging**